src/fights.ts

```diff
@@ -1916,7 +1914,8 @@ function getBestItemStealZone(): ItemStealZone | null {
         get("olfactedMonster") === zone.monster ||
         get("_gallapagosMonster") === zone.monster)
   );
-  const vorticesAvail = Math.floor(get("_fireExtinguisherCharge") / 10);
+  const canExtinguish = have($item`industrial fire extinguisher`);
+  const vorticesAvail = canExtinguish ? Math.floor(get("_fireExtinguisherCharge") / 10) : 0;
```

Comment:
```suggestion
  const vorticesAvail = have($item`industrial fire extinguisher`) ? Math.floor(get("_fireExtinguisherCharge") / 10) : 0;
```
Supes optional:
Deleting the canExtinguish line. Feels more stylistically similar to hugsAvail. Alternately, you can make a canHug variable.